AttitudeManager/Src/AM.cpp

```diff
@@ -19,6 +19,19 @@ AttitudeManagerInput AttitudeManager::control_inputs = {
     .throttle = 0.0f
 };
 
+AxisPIDs axisPIDs = {
```

Comment:
You should be able to declare these in the AM::fbwa class (same ideas as prev. comments)

You shouldn't have them in the main AM.cpp at all, since there will be more flight modes in the future which will also have varying different PID's.